olp-cpp-sdk-core/tests/client/OlpClientTest.cpp

```diff
@@ -2162,4 +2162,18 @@ TEST_F(OlpClientMergeTest, NoMergeMultipleCallbacks) {
   }
 }
 
+TEST_P(OlpClientTest, UrlWithoutProtocol) {
```

Comment:
Test missing for the async CallApi().

Reply:
OlpClientTest uses test with parameters,
```
class OlpClientTest : public ::testing::TestWithParam<CallApiType> {
 protected:
  void SetUp() override {
    network_ = std::make_shared<NetworkMock>();
    client_settings_.network_request_handler = network_;
    switch (GetParam()) {
      case CallApiType::ASYNC:
        call_wrapper_ =
            std::make_shared<CallApiGeneric<CallApiType::ASYNC>>(client_);
        break;
      case CallApiType::SYNC:
        call_wrapper_ =
            std::make_shared<CallApiGeneric<CallApiType::SYNC>>(client_);
        break;
      default:
        ADD_FAILURE() << "Invalid type of CallApi wrapper";
        break;
    }

```

Reply:
Ahh, yes. Forgot about that.